nautobot/docs/rest-api/overview.md

```diff
@@ -283,7 +283,7 @@ If we wanted to assign this IP address to a virtual machine interface instead, w
 
 Most API endpoints support an optional "brief" format, which returns only a minimal representation of each object in the response. This is useful when you need only a list of available objects without any related data, such as when populating a drop-down list in a form. As an example, the default (complete) format of an IP address looks like this:
 
-```
+```json
 GET /api/ipam/prefixes/7d2d24ac-4737-4fc1-a850-b30366618f3d/
 
 {
```

Comment:
Sure if you prefer.  I personally thought the syntax highlighting from the `json` tag made it easier to read.
The other sections further down the page, such as in the pagination section, have json formatting already in there.

Thoughts?

Reply:
I was suggesting applying `no-highlight` only to the first line as a separate block, since it's not actually part of the JSON, then keeping `json` for the actual response body. Sorry that was unclear!

Reply:
Ah gotcha.  Thank you!